doc/model/train-energy-hessian.md

```diff
@@ -0,0 +1,48 @@
+# Fit energy Hessian
+
+To train a model that takes Hessian matrices, i.e., the second order derivatives of energies w.r.t coordinates as input, you only need to prepare full Hessian matrices and modify the `loss` section to define the Hessian-specific settings, keeping other sections the same as the normal energy model's input script.
+
+Note that fitting energy Hessians is only supported in the **PyTorch** backend as of now.
+
+## Energy Hessian Loss
+
+If you want to train with Hessians, you are expected to add the start and limit prefactors of Hessians, i.e.,`start_pref_h` and `limit_pref_h` to the `loss` section in the `input.json`:
```

Comment:
Please add links for options like below.

https://github.com/deepmodeling/deepmd-kit/blob/c2944eb37a736ffd4c51b697fefe5518d3653c2a/doc/model/train-energy.md?plain=1#L92